services/movie.py

```diff
@@ -1,14 +1,19 @@
+import init_django_orm  # noqa: F401
 from django.db.models import QuerySet
-
+from django.db.transaction import atomic
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        title: str = None,
```

Comment:
If argument can be `None` you must show it in type hint. Fix everywhere.
```suggestion
        title: str | None = None,
```